### PR TITLE
Make all Taglines use imperative mood

### PR DESCRIPTION
**Pull request by JesseWeinstein**
A much smaller PR than #3018 . See discussion in https://github.com/ipfs/community/pull/150

### CODE REVIEW

**Comment by whyrusleeping** [comment]
cc @RichardLitt for review

File: core/commands/bitswap.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ import (
 
 var BitswapCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "A set of commands to manipulate the bitswap agent.",
+		Tagline:          "Interact with the bitswap agent.",
```
Comment (RichardLitt) [inline core/commands/bitswap.go]: Not sure this is a useful change; bitswap isn't a command, it is a set of subcommands.

File: core/commands/block.go [hunk 2]
```diff
@@ -26,7 +26,7 @@ func (bs BlockStat) String() string {
 
 var BlockCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Manipulate raw IPFS blocks.",
+		Tagline: "Interact with raw IPFS blocks.",
```
Comment (RichardLitt) [inline core/commands/block.go]: Why the change?

File: core/commands/files/files.go [hunk 3]
```diff
@@ -24,7 +24,7 @@ var log = logging.Logger("cmds/files")
 
 var FilesCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Manipulate unixfs files.",
+		Tagline: "Interact with unixfs files.",
```
Comment (RichardLitt) [inline core/commands/files/files.go]: Why the change?

File: core/commands/object/object.go [hunk 4]
```diff
@@ -293,7 +293,7 @@ var ObjectStatCmd = &cmds.Command{
 
 var ObjectPutCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Stores input as a DAG object, outputs its key.",
+		Tagline: "Store input as a DAG object, print its key.",
```
Comment (RichardLitt) [inline core/commands/object/object.go]: Print is a subset of output, I would think. Could probably stay with output.

File: core/commands/bitswap.go (hunk 1 above)
Comment (whyrusleeping) [inline core/commands/bitswap.go]: I think this is fine, the previous tag wasnt imperative

File: core/commands/tar.go [hunk 5]
```diff
@@ -13,7 +13,7 @@ import (
 
 var TarCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Utility functions for tar files in IPFS.",
+		Tagline: "Interact with tar files in IPFS.",
```
Comment (RichardLitt) [inline core/commands/tar.go]: Same here: ipfs tar is a set of functions, and can't manipulate. Declarative syntax is fine.

File: core/commands/tour.go [hunk 6]
```diff
@@ -14,7 +14,7 @@ import (
 
 var tourCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "An introduction to IPFS.",
+		Tagline: "Provide an introduction to IPFS.",
```
Comment (RichardLitt) [inline core/commands/tour.go]: Not necessary. "Introduce IPFS" might be better.

**Comment by RichardLitt** [comment]
Overall looks good; some comments.

File: core/commands/block.go (hunk 2 above)
Comment (JesseWeinstein) [inline core/commands/block.go]: Consistency -- using the same verb for all the toplevel commands that contain other subcommands.

File: core/commands/object/object.go (hunk 4 above)
Comment (JesseWeinstein) [inline core/commands/object/object.go]: I agree it's a subset, but it seemed like a useful distinction to make.

Comment (RichardLitt) [inline core/commands/object/object.go]: I'm not sure. Print feels like it should mean stderr, to me, but I think this is just stdout. @whyrusleeping?

Comment (whyrusleeping) [inline core/commands/object/object.go]: i think print is fine

File: core/commands/tour.go (hunk 6 above)
Comment (RichardLitt) [inline core/commands/tour.go]: Fix this and we're good to merge?

**Comment by whyrusleeping** [comment]
@RichardLitt go ahead and tag this as RFM when you think its ready

**Comment by RichardLitt** [comment]
RFM.

**Comment by whyrusleeping** [comment]
wait what? the comments werent addressed yet

File: core/commands/bitswap.go (hunk 1 above)
Comment (RichardLitt) [inline core/commands/bitswap.go]: @whyrusleeping thinks it's fine. Works for me.

File: core/commands/block.go (hunk 2 above)
Comment (RichardLitt) [inline core/commands/block.go]: @whyrusleeping thinks it's fine. Works for me.

File: core/commands/files/files.go (hunk 3 above)
Comment (RichardLitt) [inline core/commands/files/files.go]: @whyrusleeping thinks it's fine. Works for me.

File: core/commands/object/object.go (hunk 4 above)
Comment (RichardLitt) [inline core/commands/object/object.go]: @whyrusleeping thinks it's fine. Works for me.

File: core/commands/tar.go (hunk 5 above)
Comment (RichardLitt) [inline core/commands/tar.go]: @whyrusleeping thinks it's fine. Works for me.

File: core/commands/tour.go (hunk 6 above)
Comment (RichardLitt) [inline core/commands/tour.go]: Actually, this is more accurate than my suggestion. Works for me.

**Comment by RichardLitt** [comment]
@whyrusleeping See comments. This works for me.
